modules/core/src/main/java/org/apache/fluo/core/async/SyncCommitObserver.java

```diff
@@ -28,27 +30,32 @@
   private volatile boolean aacked = false;
   private volatile Exception error = null;
   private volatile String commitFailMsg = "";
+  private AtomicBoolean anyMethodCalled= new AtomicBoolean(false);
```

Comment:
This line doesn't conform to our coding standards.  However, don't worry about fixing it by hand. Fluo has a Maven plugin that formats our code. It runs when you run `mvn verify`.  Can you run this, commit any changes it makes to this file, and push this commit to your branch?  It will update this pull request.

Reply:
@asr21 if you have problems running `mvn verify`, then just running `mvn compile` should format the code.